src/db_state.rs

```diff
@@ -244,6 +251,8 @@ impl CoreDbState {
             next_wal_sst_id: 1,
             last_compacted_wal_sst_id: 0,
             last_l0_clock_tick: i64::MIN,
+            last_l0_seq: 0,
+            last_seq: 0,
```

Comment:
I'd prefer we keep the value that we track on a per-write basis in `DbInner` rather than in `CoreDbState`. `CoreDbState` is meant to be the in-memory representation of the db's manifest.

Reply:
how about put it in `DbState` (in b9481db)? i think this is indeed a state..

and how about renaming CoreDbState to something like DbManifest to reflect that it's an in-memory manifest, to differentiate it from other states?

Reply:
Yeah putting it in `DbState` makes sense for the seq num. I think the name change makes sense too, but probably in another PR.